packages/engine/src/avatar/components/NormalizedBoneComponent.ts

```diff
@@ -0,0 +1,37 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { defineComponent, S } from '@ir-engine/ecs'
+import { Bone } from 'three'
+
+export const NormalizedBoneComponent = defineComponent({
+  name: 'NormalizedBoneComponent',
+
+  schema: S.Required(S.Type<Bone>()),
+
+  onSet: (entity, component, mesh: Bone) => {
```

Comment:
i think this is redundant with schemas?

Reply:
Yes that's right, left this in by mistake when copy/pasting BoneComponent. It's still present somewhat redundantly there but it still throws an error. Going to remove the redundant set from both places.